### ISSUE TITLE
Error when loading any model

### ISSUE THREAD

**Issue by GentlemanOfCulture** [oobabooga/text-generation-webui#5713]
### Describe the bug

I get an error upon trying to load any model, no matter the type. Exl2, GPTQ, etc...

I am using the one-click installation. The last time I ran the program and updated was just over a week ago, so I was using a relatively recent version before I updated today.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

I updated the program in the usual way, through the bat file. Afterwards, I got an error after trying to load a model.

### Logs

```shell
File "C:\Users\Zugzwang\Desktop\test\text-generation-webui\modules\ui_model_menu.py", line 245, in load_model_wrapper


shared.model, shared.tokenizer = load_model(selected_model, loader)

                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\Zugzwang\Desktop\test\text-generation-webui\modules\models.py", line 87, in load_model


output = load_func_map[loader](model_name)

         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\Zugzwang\Desktop\test\text-generation-webui\modules\models.py", line 378, in ExLlamav2_HF_loader


from modules.exllamav2_hf import Exllamav2HF
File "C:\Users\Zugzwang\Desktop\test\text-generation-webui\modules\exllamav2_hf.py", line 7, in


from exllamav2 import (
ImportError: cannot import name 'ExLlamaV2Cache_Q4' from 'exllamav2' (C:\Users\Zugzwang\Desktop\test\text-generation-webui\installer_files\env\Lib\site-packages\exllamav2_init_.py)
```

### System Info

```shell
-Geforce RTX 3060
-16 gigs of system ram
```

**Comment by GentlemanOfCulture** [comment]
I'm using windows 10, if that matters.

**Comment by wwithoutn** [comment]
I am not a pro here but mostly this will happen with you have not enough VRAM or System RAM.

Can you provide 1-2 models name that failed to load in?

**Comment by p1nkl0bst3r** [comment]
I looks like you are not using the latest exllamav2 version. You should be using version 0.0.15

**Comment by GentlemanOfCulture** [comment]
Thanks for your responses. It wasn't VRAM, as I was using models that I have been loading for months. The answer was simple - I just needed to update again through a different BAT file.

**Here is what happened:**
During the previous update, it deleted the old BAT file that I normally use to update the program and created a new BAT file called "update_wizard_windows.bat", which did not exist before. It then closed the window. I had assumed that the update completed, but evidently that was not the case. After running the newly created "update_wizard_windows.bat", everything is working again.

This has been resolved!
